src/signTx.c

```diff
@@ -520,6 +520,7 @@ static void signTx_handleInitAPDU(uint8_t p2, const uint8_t* wireDataBuffer, siz
 			uint8_t numOutputs[4];
 			uint8_t numCertificates[4];
 			uint8_t numWithdrawals[4];
+			uint8_t numReferenceInputs[4];
 			uint8_t numWitnesses[4];
```

Comment:
This should be the last item in the list, possibly separated by a blank line, since it's not part of tx body.